lib/memory_profiler.js

```diff
@@ -1,103 +0,0 @@
-const chalk = require('chalk')
```

Comment:
Happy to put this back in, but decided to remove for now instead of commenting out.